packages/dbus-broker/dbus-broker.spec

```diff
@@ -39,6 +52,7 @@ CONFIGURE_OPTS=(
  -Ddocs=false
  -Dlauncher=true
  -Dselinux=true
+ -Dcatalogdir=%{_cross_journalcatalogdir}
```

Comment:
Why is this a required option now when it wasn't before?

Reply:
It is automatically set when `-Dlauncher=true`. Once we stop building the launcher, this config won't bet set. This is just to prepare the build for when the launcher isn't build anymore.